buildozer/__init__.py

```diff
@@ -311,7 +311,7 @@ def cmd(self, command, **kwargs):
                     ret_stdout.append(chunk)
                 if show_output:
                     if IS_PY3:
-                        stdout.write(str(chunk))
+                        stderr.write(chunk.decode('utf-8'))
```

Comment:
well I only see this now as I'm working for https://github.com/kivy/buildozer/issues/857
But I think we should not be writing to `stderr` here right? Feels like a failed copy paste

Reply:
Indeed...

Reply:
Thanks for confirming I was still doubting. I'm working on a unit test now for #857 it's almost done so I'll address that as well.

Reply:
Addressed in https://github.com/kivy/buildozer/pull/905